conjur/init/init_controller.py

```diff
@@ -98,7 +98,7 @@ def get_account_info(self, conjurrc_data):
             # pylint: disable=broad-except,logging-fstring-interpolation
             except Exception as error:
                 # pylint: disable=line-too-long,logging-fstring-interpolation
-                logging.debug(f"Unable to fetch the account from the Conjur server. Reason: {error}")
+                logging.warning(f"Unable to fetch the account from the Conjur server. Reason: {error}")
```

Comment:
```suggestion
                logging.warning(f"Failed to fetch the account from the Conjur server. Reason: {error}")
```

Reply:
I purposefully didn't want to say "Failed" because we are discovering the account and it is possible that the user is running in OSS so that ability to send a request to /info to discover the account is not an option.

Reply:
So this shouldn't be a warning.
Make it an info.